crates/control/src/referee_pose_detection_filter.rs

```diff
@@ -1,32 +1,30 @@
 use std::{
-    collections::BTreeMap,
-    sync::Arc,
+    collections::{BTreeMap, VecDeque},
     time::{Duration, SystemTime},
 };
 
 use color_eyre::Result;
 use context_attribute::context;
-use coordinate_systems::{Field, Ground};
 use framework::{AdditionalOutput, MainOutput, PerceptionInput};
 use hardware::NetworkInterface;
-use linear_algebra::Isometry2;
 use serde::{Deserialize, Serialize};
-use spl_network_messages::{HulkMessage, PlayerNumber};
+use spl_network_messages::PlayerNumber;
 use types::{
-    cycle_time::CycleTime,
-    fall_state::FallState,
-    messages::{IncomingMessage, OutgoingMessage},
-    players::Players,
+    cycle_time::CycleTime, fall_state::FallState, messages::IncomingMessage, players::Players,
     pose_kinds::PoseKind,
 };
 
 #[derive(Deserialize, Serialize)]
 pub struct RefereePoseDetectionFilter {
     detection_times: Players<Option<SystemTime>>,
+    own_detected_above_arm_poses_queue: VecDeque<bool>,
```

Comment:
Why `own_...`? Can that be omitted?

Reply:
This was because we distinguish between own detections of the referee and the detections of the other robots (received via a message). I think this can be omitted.

Done